### PR TITLE
Include version in dir

### PR DESCRIPTION
**Pull request by percona-csalguero**
I would like to use sanbox package in a project where I need to start a sandbox to test queries.
The project is: [minimum_permissions](https://github.com/Percona-Lab/minimum_permissions) (still work in progress)
Since I don't control the queries than can be tested and some queries like a CREATE TABLE have an implicit autocommit,I need a sandbox.

The only limitation I found to use the sandbox package is that in some functions, like `CreateSingleSandbox`, there is a line that says:
```
sdef.Basedir = sdef.Basedir + "/" + sdef.Version
```
and that forces me to have the MySQL binaries in path including the version number like `/path/to/mysql/binaries/<mysql version>` but, I would like to be able to use existing MySQL binaries.
The change I am proposing appends the MySQL version in the main BaseDir so functions like `CreateSingleSandbox` will receive the complete `BaseDir` and any directory having the MySQL binaries can be used to start a sandbox.

Basically, what I am proposing is to append the version to the binaries dir in the `FillSdef` function (cmd/single.go) and removing all `+ "/" + sdef.Version` in other places.

There is only a few really necessary changes and I apologize for the include reordering and some other intentations or blanks removed but thos were automatically made by go format. If you prefer, I can revert
those changes.

Regards

### CODE REVIEW

**Comment by percona-csalguero** [comment]
Hi. 
I've added the missing comma that was causing the error (I merged the latest master branch and I didn't properly check it).
The use case is this.
I already have MySQL binaries (the ones I use to test the toolkit) for different versions in my home, in the mysql dir (`${HOME}/mysql`)
The output of a `tree` shows this:
```
tree -L 1
.
├── my-5.5
├── my-5.6
├── my-5.7
├── my-8.0
├── ps-5.5
├── ps-5.6
├── ps-5.7
└── tars
```
I would like to use only the `sandbox` package from dbdeployer.
If I calll `CreateSingleSandbox`, passing a `SandboxDef` like this one: 
```
sd := SandboxDef {
     BaseDir: "/home/me/mysq/my-5.7", 
     Version: "5.7.21", 
     .
     .
     .
}
```
`CreateSingleSandbox` will add the version to the BaseDir, and it will look for the binaries at `/home/me/mysq/my-5.7/5.7.21` instead of `/home/me/mysq/my-5.7/` where the binaries I want to use are.

By appending the version **before** calling `CreateSingleSandbox` (or other functions) my intention is to achieve 2 things:
1) Being able to use MySQL binaries from any directory
2) Populate `BaseDir` in only one place instead of adding `+ "/" + sdef.Version` in more than one place.

Thanks for taking the time to review this.

**Comment by datacharmer** [comment]
Oh, now it's clear. I admit that when I designed dbdeployer I didn't think of this usage.
I will run a the existing  tests to see whether this change affects the behavior, and then I will merge (It will probably happen during the week end.)

I should also think of tests for this use case, i.e. creating sandboxes without dbdeployer binaries.
I think that I should consider how to use/modify the defaults without the CLI interface.
Thanks for providing this interesting use case!

**Comment by datacharmer** [comment]
Release 1.4.1 includes coding samples and some documentation. (./docs/coding)
Notice that the call to CreateSingleSandbox was changed: an unused parameter was removed)
